source/manual/zendesk.html.md

```diff
@@ -0,0 +1,88 @@
+---
+owner_slack: "#govuk-2ndline"
+title: Zendesk
+parent: "/manual.html"
+layout: manual_layout
+section: 2nd line
+type: learn
+last_reviewed_on: 2019-08-16
+review_in: 6 months
+---
+
+2nd line Zendesk tickets are technical errors reported by our users - including government publishers -
+and general queries from users of data.gov.uk, for whom 2nd line provides 1st line support.
+
+## Get started
+
+[Create an account](zendesk-create-account), then ask a fellow 2nd liner to assign a new ticket to
+"2nd/3rd Line—Zendesk Administration" asking to give you access to "2nd Line - GOV.UK Alerts and Issues".
+
+When you're logged in, you should be looking at the ['2nd Line--GOV.UK Alerts and Issues' queue][zendesk-queue].
+
+## Priorities
+
+Work on 'High' priority tickets first - these will have been triaged by 2nd line already, so should be
+relevant and actionable - and start with the oldest by date. If external assistance from outside 2nd
+line is required, assign them on the ticket and move on to the next ticket (keeping the ticket in the
+2nd line queue).
+
+Ignore 'Low' priority tickets - these have been triaged by 2nd line for GOV.UK product managers to review.
+
+New tickets are 'Normal' priority by default and require triaging (i.e. a developer reading and categorising
+it).
+
+## Triaging a ticket
+
+If you have any doubts as to the legitimacy or urgency of a ticket, it should be deferred to the delivery manager.
+
+Once you've established that a ticket is legitimate, you should follow [the process in this diagram](zendesk-triage-diagram).
+There is a printout on the 2nd line desk. In general:
```

Comment:
Since we're linking to the flowchart on the line above, do we need to summarise the content here?  It risks this list getting out of alignment with the diagram, or for developers to not realise there are alternative routes of escalation (e.g. to RE) which are not listed here.